### PR TITLE
BaSP-TG: Added get super admins list 

### PR DESCRIPTION
**Pull request by LucianaPrat**
Added method to get super admins, filtering by first name.
[localhost:3000/super-admins/filter?firstName=Effie](http://localhost:3000/super-admins/filter?firstName=Effie)

### CODE REVIEW

**Review by joryapur — COMMENTED** [review]
I think everything is fine! Thank you for doing the group work and collaborating

File: src/resources/super-admins.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+
+const router = express.Router();
+
+const superAdmins = require('../data/super-admins.json');
+
+router.get('/filter', (req, res) => {
```
Comment (nicolasblasco) [inline src/resources/super-admins.js:7]: We should change the route to /:firstName

File: src/resources/super-admins.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const { firstName } = req.query;
+  const foundSuperAdmins = superAdmins.filter((superAdmin) => superAdmin.firstName === firstName);
+  if (foundSuperAdmins) {
+    res.status(200).send({ success: true, msg: 'Super Admin found successfully', data: foundSuperAdmins });
```
Comment (nicolasblasco) [inline src/resources/super-admins.js:11]: The response should send JSON instead of send

File: src/resources/super-admins.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  } else {
+    res.status(404).send({ success: false });
```
Comment (nicolasblasco) [inline src/resources/super-admins.js:13]: Same as above

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

**Review by nicolasblasco — CHANGES_REQUESTED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**